dbms/src/TiDB/Raft/RaftLogManager.cpp

```diff
@@ -0,0 +1,184 @@
+// Copyright 2023 PingCAP, Ltd.
```

Comment:
Why not still put these files in Transaction?

Reply:
there are too many codes that mix Raft, RegionMeta, codec under the Transaction folder. I want to split it into different subfolders.
But sure we can do it later.

Reply:
Totally agree for the first part, the Transaction folder is too messy. I think it is the time we divide the folder into several sub folders.
However, I think `KVStore/Transaction` and `KVStore/Raft` is better than `TiDB/Raft`, since raft and transaction is in TiKV(KVStore) layer, rather than in TiDB layer.